src/Savegame/BattleItem.cpp

```diff
@@ -1316,6 +1316,19 @@ std::string debugDisplayScript(const BattleItem* bt)
 	}
 }
 
+void getActionTUsScript(const BattleItem* bt, const BattleUnit* bu, const int battle_action, int& i)
```

Comment:
Small change, usually return value is second argument, see `getArmorMaxScript` or `reduceByResistanceScript`.
This is because real first argument is moved to "function name" and second become first.